1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,20 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
 ];
 
+for (let i = 0; i < tubeStations.length; i++) {
+  console.log(tubeStations[i]);
+}
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+
+for (let i = 0; i < str.length; i++) {
```

Comment:
Works perfectly fine! 
Using the same comment above though, attempt this with the `for-of` loop